app/components/learnergroup-user-manager.hbs

```diff
@@ -31,23 +31,15 @@
                 {{/if}}
               </th>
               <SortableTh
-                @colspan={{2}}
-                @click={{fn this.setSortBy "firstName"}}
+                @colspan={{4}}
```

Comment:
First click here doesn't do anything other than update the URL. It should reverse the sort.  Also the icon here doesn't match the one on users in cohort not in this group. Should be AZ in both places.

Reply:
> First click here doesn't do anything other than update the URL. It should reverse the sort. Also the icon here doesn't match the one on users in cohort not in this group. Should be AZ in both places.

i fixed the sort order bug by updating the default sort order in the controller.

Reply:
fixed, the correct icon now gets applied. there is a bug in the `<SortableTh>` component where the sort order icon does not get updated if sort order is reversed, but this will need to be corrected in common.